### PR TITLE
ON-6735 # Add generate page element function

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: lib/generate-page-element.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+) /* :PageElement */ {
+  if (!formElementGenerationData) {
+    throw new Error(
+      'no page element generation data provided, please provide this data to generate a page element',
+    )
+  }
+  const formElement /* : Object */ = {
+    ...formElementGenerationData,
+    type: 'page',
+    id: uuid(),
```
Comment (mymattcarroll) [inline lib/generate-page-element.js:19]: I think we want to allow them to have their own `id` if they want.

Comment (kizaonline) [reply lib/generate-page-element.js:19]: I agree but the generateFormElement also sets the id

Comment (kizaonline) [reply lib/generate-page-element.js:19]: Happy to change both though since its not really a breaking change

**Comment by mymattcarroll** [comment]
👍  LGTM

**Pull request merged**